pack/tcu/wos_encounter.json

```diff
@@ -0,0 +1,811 @@
+		"illustrator": "Jose Vega",
+		"name": "Chapel Crypt",
+		"pack_code": "wos",
+		"position": 173,
+		"quantity": 1,
+		"shroud": 6,
+		"text": "While there are no ready enemies at Chapel Crypt, it gets -3 shroud.",
+		"type_code": "location"
+	},
+	{
+		"clues": null,
+		"code": "05174b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 14,
+		"faction_code": "mythos",
+		"illustrator": "Jose Vega",
+		"hidden": true,
+		"name": "Chapel Crypt",
+		"pack_code": "wos",
+		"position": 174,
+		"quantity": 1,
+		"shroud": 6,
+		"text": "While there are no [[Hex]] treacheries among investigators' threat areas at Chapel Crypt, it gets -3 shroud.\n<b>Haunted</b> - Find the topmost [[Hex]] treachery in the standard encounter discard pile and put it into play in your threat area.",
+		"traits": "Spectral.",
+		"type_code": "location"
+	},
+	{
+		"clues": 0,
+		"code": "05174",
+		"back_link": "05174b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 14,
+		"faction_code": "mythos",
+		"flavor": "A narrow stone staircase guides you deep below the chapel, where those of good repute were given “proper” burials. In a far corner of the room, a strange, dusty contraption is connected to one of the coffins, as dead as everything else in this room.",
+		"illustrator": "Jose Vega",
+		"name": "Chapel Crypt",
+		"pack_code": "wos",
+		"position": 174,
+		"quantity": 1,
+		"shroud": 6,
+		"text": "While there are no [[Hex]] treacheries among investigators' threat areas at Chapel Crypt, it gets -3 shroud.",
+		"type_code": "location"
+	},
+	{
+		"clues": null,
+		"code": "05175b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 15,
+		"faction_code": "mythos",
+		"illustrator": "Dual Brush Studios",
+		"hidden": true,
+		"name": "Chapel Attic",
+		"pack_code": "wos",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After you draw a non-weakness card from your deck while at Chapel Attic: Place that card facedown beneath Chapel Attic, out of play.\n[reaction] After you successfully investigate Chapel Attic: Add each card beneath it to its owner's hand.\n<b>Haunted</b> - Discard a random card from beneath Chapel Attic.",
+		"traits": "Spectral.",
+		"type_code": "location"
+	},
+	{
+		"clues": 0,
+		"code": "05175",
+		"back_link": "05175b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 15,
+		"faction_code": "mythos",
+		"illustrator": "Dual Brush Studios",
+		"name": "Chapel Attic",
+		"pack_code": "wos",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After you draw a non-weakness card from your deck while at Chapel Attic: Place that card facedown beneath Chapel Attic, out of play.",
```

Comment:
Please add at the end: '\n [reaction] After you successfully investigate Chapel Attic: Add each card beneath it to its owner's hand.'